packages/autocannon-ui-backend/package.json

```diff
@@ -14,9 +14,9 @@
   "dependencies": {
     "autocannon": "^7.2.0",
     "env-schema": "^5.0.0",
-    "fastify": "^3.8.0",
+    "fastify": "^4.0.2",
     "fastify-cli": "^4.0.0",
-    "fastify-cors": "^6.0.2",
+    "@fastify/cors": "^8.0.0",
```

Comment:
the fact that this package has not been moved up in the list suggests that this change was done manually instead of using the package manager. please use yarn to do these changes because otherwise there's a chance that the lockfile gets out of sync